### PR TITLE
reshape array of size 1 into shape (7,) needs cftime >=1.0.1, fixes #58

### LINKED ISSUE #58: ValueError: cannot reshape array of size 1 into shape (7,)
Issue:

<br/>
conda install -c conda-forge netcdf4
Solving environment: done

## Package Plan ##

  environment location: /home/user/miniconda3/envs/mssenv3

  added / updated specs: 
    - netcdf4

The following packages will be downloaded:

    package                    |            build
    ---------------------------|-----------------
    ca-certificates-2018.8.24  |       ha4d7672_0         136 KB  conda-forge
    netcdf4-1.4.1              |   py36h62672b6_0         2.0 MB  conda-forge
    certifi-2018.8.24          |           py36_1         139 KB  conda-forge
    openssl-1.0.2p             |       h470a237_0         3.5 MB  conda-forge
    ------------------------------------------------------------
                                           Total:         5.8 MB

The following packages will be UPDATED:

    ca-certificates: 2018.03.07-0         --> 2018.8.24-ha4d7672_0 conda-forge
    certifi:         2018.8.24-py36_1     --> 2018.8.24-py36_1     conda-forge
    netcdf4:         1.4.0-py36ha06eab4_1 --> 1.4.1-py36h62672b6_0 conda-forge
    openssl:         1.0.2p-h14c3975_0    --> 1.0.2p-h470a237_0    conda-forge
<details>

```


```
</details>

<br/>
Details about  <code>conda</code> and system ( <code>conda info</code> ):
<details>

```
     active environment : mssenv3
    active env location : /home/user/miniconda3/envs/mssenv3
            shell level : 1
       user config file : /home/user/.condarc
 populated config files : /home/user/.condarc
          conda version : 4.5.11
    conda-build version : not installed
         python version : 3.6.5.final.0
       base environment : /home/user/miniconda3  (writable)
           channel URLs : https://repo.anaconda.com/pkgs/main/linux-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/free/linux-64
                          https://repo.anaconda.com/pkgs/free/noarch
                          https://repo.anaconda.com/pkgs/r/linux-64
                          https://repo.anaconda.com/pkgs/r/noarch
                          https://repo.anaconda.com/pkgs/pro/linux-64
                          https://repo.anaconda.com/pkgs/pro/noarch
                          https://conda.anaconda.org/conda-forge/linux-64
                          https://conda.anaconda.org/conda-forge/noarch
          package cache : /home/user/miniconda3/pkgs
                          /home/user/.conda/pkgs
       envs directories : /home/user/miniconda3/envs
                          /home/user/.conda/envs
               platform : linux-64
             user-agent : conda/4.5.11 requests/2.18.4 CPython/3.6.5 Linux/4.4.0-134-generic ubuntu/16.04 glibc/2.23
                UID:GID : 1000:1000
             netrc file : None
           offline mode : False


```
</details>

I had run in the problem that 1.4.0 works but 1.4.1 of netcdf4 doesn't 

https://bitbucket.org/wxmetvis/mss/issues/366/demodata-mswms-problem

currently a downgrade to 1.4.0 solves it for me.

### PR DESCRIPTION
**Pull request by ReimarBauer**
Checklist
* [X] Used a fork of the feedstock to propose changes
* [X] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [X] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [X] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by ReimarBauer** [comment]
have to wait until cftime is done

File: recipe/meta.yaml [hunk 1]
```diff
@@ -9,7 +9,8 @@ source:
   sha256: d4fc65b98e348c39d082ab6b4b7f6d636b1b4e63bec016e5bca189fee5d46403
 
 build:
-  number: 0
+  skip: true  # [py37]
```
Comment (ocefpaf) [inline recipe/meta.yaml:12]: Argh. I missed this :unamused: 

Can you please send another PR removing this and re-rendering?
